index.html

```diff
@@ -1,9 +1,96 @@
+                 <li><a href="#"> Resources </a></li>
+                <li><a href="#"> Docs </a></li>
+                <li><a href="#"> Support </a></li>
+            </ul>
+        </nav>
+        </div>
+    
+   <!-- Bottom Nav -->
+    
+         <div class="bottom-nav">
+            <img class="npm-logo" src="images/npm-logo.png" alt="NPM Logo"/>
+            <form class="search" action="/register" method="post"> 
+                <img class="magnify" src="images/magnifying-glass.png" alt="magnifying glass"/>
+                <input type="search" placeholder="Search Penguin Magic"> 
+                <button>Search</button>
+            </form>
+            <div>
+                <a class="link-join" href="#"> Join </a>
+                <a class="link-log" href="#"> Log In </a>
+            </div>
+        </div>
+    </header>
+  
+  <!-- Top -->
+  
+    <section id="top">
+        <h3>Build amazing things</h3>
+        <article>
+            <p>Essential JavaScript development tools that help you go to market faster and build powerful applications using
+  modern open source code.</p>
+        </article>
+        <div>
+            <a class="link-top-left" href="#"> See plans </a>
+            <a class="link-top-right" href="#"> Join for free </a>
+        </div>
+    </section>
+  
+  <!-- Mid -->
+  
+    <section id="mid">
+        <img class="blackHeart" src="images/triangle-hexagon.svg"/> 
+        <h2> Bring the best of open source to your company </h2>
+        <article>
+            <p>npm is the tool used by over 11,000,000 JavaScript developers around the world. Your developers already use it. Your company depends on it. Create an Org and get more out of the tools your team already knows and loves.
+            </p>
+        </article>
+    </section>
+  
+  <!-- Bottom -->
+  
+    <section id="bottom">
+        <div>
+            <img class="bottom-image" src="images/zero-configuration.svg"  alt="Logo 1"/>
+            <h1>Zero configuration</h1>
+            <p>Create an org, add your team, and start collaborating. Nothing to configure, set up, or manage. </p>
+        </div>
+        <div>
+            <img class="bottom-image" src="images/team-management.svg" alt="Logo 2"/>
+            <h1>Team management</h1>
+            <p>Control who has access to what modules within your team namespace using straightforward team management capabilities.</p>
+        </div>
+        <div>
+            <img class="bottom-image" src="images/familiar-features.svg" alt="Logo 3"/>
+            <h1>Familiar features</h1>
+            <p>npm Orgs has 100% parity with all the public npm registry features your developers already use.</p>
+        </div>
+        <div>
+            <img class="bottom-image" src="images/npm-audit.svg" alt="Logo 4"/>
+            <h1>npm audit</h1>
+            <p>Enjoy the security auditing features built into the npm client, a zero-friction way to make open source software safer.</p>
+        </div>
```

Comment:
I would propose to wrap all of these in a div and make it another flex both. That way the #bottom flex box's direction could be column (and the "create an org" button would always stay in a row below) and this div's flex box's direction would be row.